test/controllers/content_items_controller_test.rb

```diff
@@ -346,6 +346,22 @@ class ContentItemsControllerTest < ActionController::TestCase
     assert_equal response.headers["Access-Control-Allow-Origin"], "*"
   end
 
+  test "service_sign_in_options with option param set" do
+    content_item = content_store_has_schema_example("service_sign_in", "service_sign_in")
+    path = "#{path_for(content_item)}/#{content_item['details']['choose_sign_in']['slug']}"
+
+    option = content_item['details']['choose_sign_in']['options'][0]
```

Comment:
As we have to call `content_item['details']['choose_sign_in']` a couple of times in this test, could we extract it out and assign to a variable to make it a bit easier to read?